base/complex.jl

```diff
@@ -633,7 +634,7 @@ function exp(z::Complex)
     end
 end
 
-function expm1(z::Complex{T}) where T<:Real
+function expm1(z::Complex{T}) where T
```

Comment:
The parameter of `Complex` already needs to be <:Real, so this one should be ok as is?

Reply:
I removed `<:Real` because `Complex` already only takes `Real` :p

Reply:
It's been a long-standing issue (#6383) that this method signature isn't automatically adjusted to `Complex{<:Real}` according to the declaration of `Complex`. So it's still considered good practice to put `<:Real` there.

Reply:
I see.